test/infra/infra-mysql56-single/bin/docker-mysql-post.bash

```
@@ -0,0 +1,94 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+[ -f .env ] && . .env
```

Shellcheck (0.11.0)

[info] Not following: .env was not specified as input (see shellcheck -x).

(SC1091)